PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalVaultRequest.java

```diff
@@ -65,8 +68,27 @@ public String getUserAuthenticationEmail() {
         return this.userAuthenticationEmail;
     }
 
-    String createRequestBody(Configuration configuration, Authorization authorization,
-                             String successUrl, String cancelUrl) throws JSONException {
+    /**
+     * Optional: Used to determine if the customer will use the PayPal app switch flow.
+     * Defaults to `false`.
+     * - Warning: This property is currently in beta and may change or be removed in future releases.
+     *
+     * @param enablePayPalAppSwitch - A boolean value indicating whether to enable the PayPal app switch flow.
+     */
+    public void setEnablePayPalAppSwitch(Boolean enablePayPalAppSwitch) {
```

Comment:
```suggestion
    public void setEnablePayPalAppSwitch(boolean enablePayPalAppSwitch) {
```
Good to use a primitive here as well since the property is also a primitive.

Reply:
You're right, thanks!!! [76f7e00](https://github.com/braintree/braintree_android/pull/1083/commits/76f7e0011ccdf3a848252858b2327c16817e8536)